install_files/ansible-base/roles/tor-hidden-services/tasks/install_tor.yml

```diff
@@ -1,35 +1,35 @@
 ---
-- name: Install Tor project GPG signing key.
+- name: Remove Tor project GPG signing key.
   apt_key:
-    state: present
-    data: "{{ lookup('file', 'tor-signing-key.pub') }}"
+    state: absent
+    id: A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89
   tags:
     - apt
     - tor
 
 - name: Setup Tor apt repo.
   apt_repository:
-    repo: deb http://deb.torproject.org/torproject.org {{ ansible_lsb.codename }} main
+    filename: tor_apt_freedom_press
+    repo: deb {{ tor_apt_repo_url }} {{ ansible_lsb.codename }} main
```

Comment:
This logic should be removed eventually since it is duplicating the logic of the new `securedrop-config` package. I'm not going to block on it for this review and instead I'll spin  up a new ticket.